sgf-viewer/src/content.ts

```diff
@@ -0,0 +1,97 @@
+                "adversary": "34.1 million training steps, 600 visits",
+                "victim": "Latest (cp505-v1-MCTS), no search",
+                "description": [
+                    "Even without search, Katago's <code>Latest</code> network plays at the <a href=\"https://arxiv.org/pdf/2211.00241.pdf#page=16\">strength of a top-100 European professional</a>. Yet our adversary achieves a 99% win rate against this victim by playing a counterintuitive strategy. The adversary stakes out a minority territory in the corner, allowing KataGo to stake the complement, and placing weak stones in KataGo’s stake.",
+                    "KataGo predicts a high win probability for itself and, in a way, it’s right—it would be simple to capture most of the adversary’s stones in KataGo’s stake, achieving a decisive victory. However, KataGo plays a pass move before it has finished securing its territory, allowing the adversary to pass in turn and end the game. This results in a win for the adversary under the standard <a href=\"https://tromp.github.io/go.html\">Tromp-Taylor</a> ruleset for computer Go, as the adversary gets points for its corner territory (devoid of victim stones) whereas the victim does not receive points for its unsecured territory because of the presence of the adversary’s stones."
+                ]
+            },
+            {
+                "title": "KataGo with 64 visits (level of top world professionals)",
+                "dir_name": "64_visits",
+                "server": "dqn.ist.berkeley.edu",
+                "_path_comment": "Computed by `grep -l v=1024 * | head -n 10` in /nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs, see https://www.notion.so/chaiberkeley/match-cp505-64-vs-adv-1_to_8192-39e0e303cd3f45199f4ce638c92556a5 for experiment log",
+                "paths": [
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/033FE05C5431ED13.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/048A6592BB515609.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/0FF3484FD5147146.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/14F7EEBACD86547C.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/16E93C54A1C3773E.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/187A346B3CC632E1.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1CEF0F7800D4E196.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1DF2F4ED5B859942.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1EEA60F118BBB86C.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/2172599E22680D06.sgfs"
+                ],
+                "paths_1024": [
```

Comment:
what does this do?

Reply:
I'm a bit confused by this. @AdamGleave It looks like these games are 1024 and are effectively commented out by this unused key, `paths_1024`. The games we actually want are 8192 are correctly under `paths`. So I think I'll just delete everything in `paths_1024`.